### PR TITLE
Back to Apache Maven JLink Plugin

### PR DESCRIPTION
**Pull request by HGuillemet**
Some explanations about the module path and graph construction using the apache plugin could be worth added in the README.
Tell me if you'd like me to add  such explanation.

### CODE REVIEW

File: opencv-stitching-jlink/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Sample project for using OpenCV from a generated Java binding made with [Bytedeco's](http://bytedeco.org/) JavaCPP technology.
- * A jlink version of the [original stitching sample project](../opencv-stitching), using the maven-jlink plugin.
+ * A jlink version of the [original stitching sample project](../opencv-stitching), using the Apache Maven JLink plugin.
 
 On the command line, run:
 
 ```bash
-$ mvn clean package -Djavacpp.platform.custom -Djavacpp.platform.host
-$ ./target/image/bin/stitch panorama_image1.jpg panorama_image2.jpg --output panorama_stitched.jpg
+$ mvn clean package -Djavacpp.platform.host
```
Comment (saudet) [inline opencv-stitching-jlink/README.md:7]: We should leave `-Djavacpp.platform.custom` here. `-Djavacpp.platform.host` won't always work correctly without it.

Comment (HGuillemet) [reply opencv-stitching-jlink/README.md]: Re-added.
But for my information, what can turn wrong without it ?

Comment (saudet) [reply opencv-stitching-jlink/README.md]: The profiles for the `javacpp.platform` one don't get disabled without it:
https://github.com/bytedeco/javacpp-presets/wiki/Reducing-the-Number-of-Dependencies

**Comment by saudet** [comment]
If people start asking questions, I suppose we could add explanations here, but there's probably a better place for those...

**Comment by HGuillemet** [comment]
> If people start asking questions, I suppose we could add explanations here, but there's probably a better place for those...

A wiki page in javacpp repo on "JavaCPP and JPMS" ?

**Pull request merged**